### DISCUSSION TITLE
JavaDoc for Pi4j v2

### DISCUSSION THREAD

**Discussion by githopper** [Pi4J/pi4j#267] [General]
Hi Pi4J team,
I am looking for the JavaDoc for v2. I only find the older JavaDoc for v1 https://pi4j.com/1.4/apidocs/index.html

On the website it was mentioned v2 is completely different so I guess the older JavaDoc will not help a lot. I just wanted to get a brief overview about the API as I am new to Raspberry Pi and Pi4J.

**Comment by taartspi** [comment]
Good day.

From the down load page: https://pi4j.com/download/

Latest Release: [pi4j-2.3.0.zip (2.48 MiB, 2023-02-03 15:54)](https://github.com/Pi4J/download/raw/main/pi4j-2.3.0.zip)

Inside there is the javadoc jar.

Tom
